queries/c/indents.scm

```diff
@@ -1,28 +1,74 @@
 [
-  (compound_statement)
-  (preproc_arg)
```

Comment:
is `preproc_arg` related to this change? If I remember correctly it is for indenting second lines of macros. Not sure if this is tested in the tests.

Reply:
Somewhere below of this file, someone marks (preproc_arg) as @ignore, so I removed it. 
Can add it back if you think is needed

Reply:
https://github.com/nvim-treesitter/nvim-treesitter/blob/master/queries/c/indents.scm#L42
It is marked as `@ignore` right afterwards, which will cancel the `@indent`, so I removed it